gpytorch/lazy/batch_repeat_lazy_tensor.py

```diff
@@ -35,18 +35,20 @@ def __init__(self, base_lazy_tensor, batch_repeat=torch.Size((1,))):
         self.batch_repeat = batch_repeat
 
     @cached(name="cholesky")
-    def _cholesky(self):
-        res = self.base_lazy_tensor._cholesky()
+    def _cholesky(self, upper=False):
+        from .triangular_lazy_tensor import TriangularLazyTensor
+
+        res = self.base_lazy_tensor._cholesky(upper=upper)
         res = res.repeat(*self.batch_repeat, 1, 1)
-        return res
+        return TriangularLazyTensor(res, upper=upper)
```

Comment:
Do we want this? This is creating probably a TriangularLazyTensor(BatchRepeatLT(TriangularLT))

Reply:
```suggestion
        return TriangularLazyTensor(res._tensor, upper=upper)
```

Reply:
I guess I could just call the repeat on the `_tensor` directly. Amounts to the same thing, but might be less convoluted.